### ISSUE TITLE
scripts/create_baseline_stubs.py instructions are outdated

### ISSUE THREAD

**Issue by Akuli** [python/typeshed#8837]
I would prefer:
 ```
 1. Manually review the generated stubs in {stub_dir}
 2. Run tests locally if you want (see CONTRIBUTING.md)
 3. Commit the changes on a new branch and create a typeshed PR
 ```

The CI will check everything anyway, and is set up so that you don't have to run anything locally. This would also be consistent with CONTRIBUTING.md.

_Originally posted by @Akuli in https://github.com/python/typeshed/issues/8686#issuecomment-1237374669_

**Comment by srittau** [comment]
One problem with the generated stubs were overzealous re-exports. Is this still an issue? In that case we should probably mention it. But +1 from me.

**Comment by AlexWaygood** [comment]
@Avasam has a proposed fix for this included as part of #8798.

**Comment by AlexWaygood** [comment]
> One problem with the generated stubs were overzealous re-exports. Is this still an issue? In that case we should probably mention it. But +1 from me.

This is _definitely_ still an issue. But @hauntsaninja mentioned to me recently that this can probably be fixed just by passing the `--export-less` argument to stubgen. We should see if that fixes the problem — if so, there's no need to mention it to contributors in the `create_baseline_stubs.py` instructions.
